lib/semitran.gi

```diff
@@ -383,9 +383,8 @@ end);
 InstallMethod(IsomorphismTransformationSemigroup, 
 "for a transformation semigroup", 
 [IsTransformationSemigroup], 
-function(S)
-  return MagmaIsomorphismByFunctionsNC(S, S, IdFunc, IdFunc);
-end);
```

Comment:
You sure you don't mean `IdentityMapping(S)`?

Reply:
These are the lines that I'm deleting, I'm not sure what you mean.